plugins/jira/commands/update-qa-contact.md

```
@@ -0,0 +1,1189 @@
+---
+description: Update QA contact name for JIRA issues filtered by status or specific issue
+argument-hint: <status> <from-qa> <to-qa> [project] [--dry-run] | --issue-key <KEY> [<from-qa>] <to-qa> [--dry-run]
+---
+
+## Name
+jira:update-qa-contact
+
+## Synopsis
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)